Python_sem3/DZ4.py

```diff
@@ -0,0 +1,15 @@
+# Задача на семинаре
+# ✔ Создайте вручную список с повторяющимися целыми числами.
+# ✔ Сформируйте список с порядковыми номерами нечётных элементов исходного списка.
+# ✔ Нумерация начинается с единицы.
+
+
+numbers = [1, 2, 3, 4, 5, 13, 4, 42, 1, 2, 3, 4, 5, 67, 8, 89]
+numbers2 =set(numbers)
+new_list = []
+for elem in numbers2:
```

Comment:
Ни к чему исполосовывать переменную единожды
```suggestion
new_list = []
for elem in set(numbers):
```